File: seongwon/Programmers/Level_0/Lv0_간단한_식_계산하기.cpp
#include <string>
#include <vector>
#include <sstream>
using namespace std;

int solution(string binomial) {
    int answer = 0;
    int a, b;
    char op;
    stringstream st(binomial);
    st >> a >> op >> b;
    switch(op){
        case '+':
            answer = a+b;
            break;
        case '-':
            answer = a-b;
            break;
        case '*':
            answer = a*b;
            break;
        }
    
    return answer;
}

File: dohyeon/Programmers/Level_0/Lv0_간단한_식_계산하기.py
def solution(binomial):
    answer = 0
    binomial = binomial.split(" ")
    a = int(binomial[0])
    op = binomial[1]
    b = int(binomial[2])
    if op == "+" :
        answer = a + b
    elif op == "-" :
        answer = a - b
    elif op == "*" :
        answer = a * b

    return answer

print(solution("43 + 12"))